src/OpenApi/JsonSchema/Normalizer/ReferenceNormalizer.php

```diff
@@ -38,17 +38,123 @@ public function denormalize($data, $class, $format = null, array $context = [])
         if (!is_object($data)) {
             return null;
         }
+
+        $object = null;
```

Comment:
Tihs file can't be edited manualy since it's generated by the JSON Schema library, it will be overwriten on next OpenAPI schema update :/

Reply:
I saw the "do not edit" hint but still wanted to solve this and raise awareness